css-view-transitions-1/Overview.bs

```diff
@@ -795,33 +818,56 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 		To <dfn lt="capture the image|capturing the image">capture the image</dfn> given an {{Element}} |element|, perform the following steps.
 		They return an image.
 
-		1. Render the referenced element and its descendants,
-			at the same size that they would be in the document,
-			over an infinite transparent canvas with the following characteristics:
+		1. If |element| is the [=document element=], then:
+
+			1. Render the region of the [=document element=] that intersects the [=snapshot viewport=],
+				on a transparent canvas the size of the [=snapshot viewport=],
+				following the [=capture rendering characteristics=], and these additional characteristics:
+
+				- Areas outside the [=scrolling box=] should be rendered as if they were scrolled to, without moving the [=layout viewport=].
+					This must not trigger events related to scrolling or resizing, such as {{IntersectionObserver}}s.
+
+					Issue: "scrolling box" and "layout viewport" are not exported. They either need exporting, or replaced with better references.
 
-			* The origin of |element|'s [=ink overflow rectangle=] is anchored to canvas origin.
+				- Areas that cannot be scrolled to (i.e. they are out of scrolling bounds),
+					should render transparent.
```

Comment:
@bramus what's your take on this? I'm not sure if I'm using the right language.

I'm trying to say that, if the keyboard is open, and it's changed the size of the viewport, we should capture the area under the keyboard as if it were removed, but the layout viewport stayed the same size.

Reply:
The css-viewport spec speaks of [“interactive widgets”](https://drafts.csswg.org/css-viewport/#interactive-widget-section). I guess you could simply say “areas obscured by interactive widgets should render transparent”, and leave the definition up to the css-viewport spec.

Reply:
I'm not sure we want them to render transparent though. I was trying to clarify that with the "Areas outside the [=scrolling box=] should be rendered as if they were scrolled to" language.

Reply:
Ah I see. Was (incorrectly) focussing on the last line there. The way you worded things in your comment is much clearer than the text currently in the spec. 

My hunch would be to try to include a reference to the interactive widgets. Does “the presence of interactive widgets and their effects on layout are taken into account when rendering“ capture it? Possibly, explicitly also mention that this implies that areas obscured by the interactive widgets do get rendered if they are in the snapshot viewport, and that no scroll/resize events get triggered.